maplibre-adapter/src/main/java/com/amplifyframework/geo/maplibre/http/AWSRequestSignerInterceptor.kt

```diff
@@ -43,15 +45,20 @@ private fun Request.Builder.copyFrom(request: AWSRequest): Request.Builder {
     return this.url(urlBuilder.build())
 }
 
+const val AMAZON_HOST = "amazonaws.com"
+
 /**
  * Interceptor that can authorize requests using AWS Signature V4 signer.
  */
 internal class AWSRequestSignerInterceptor(
     private val plugin: AWSLocationGeoPlugin
-): Interceptor {
+) : Interceptor {
 
     override fun intercept(chain: Interceptor.Chain): Response {
         val request = chain.request()
+        if (!request.url.host.contains(AMAZON_HOST)) {
```

Comment:
this is needed because the style can be augmented after it's loaded. In case developers add external data or image sources to the style they also go through the interceptor. So the interceptor need to be applied to AWS requests only (that's how the original interceptor in the developer preview of Amazon Location Service was implemented)